firstAssignment/biodata.html

```diff
@@ -4,43 +4,38 @@
 <head>
     <meta charset="utf-8" />
     <title>biodata</title>
-    <style>
-        .biodata {
-            background-color:lightblue;
-            text-align:center;
-        }
-        .container{
-            background-color:aliceblue;
-        }
-        h3, p {
-            display: inline-block;
-        }
-        
-    </style>
+   <link rel="stylesheet" href="style.css">
 </head>
 <body>
+   
+    <div class="biodata">
+        <h1>BIODATA</h1>
+    </div>
     <div class="container">
-        <div class="biodata">
-            <br /><h1>BIODATA</h1><br /><br />
+        <div><h3>Name :</h3>
+            <p>Umesh U I</p>
```

Comment:
align this correctly, so the code will be in more readable format. The h3 tag should be under the div tag and properly aligned